docs/overview/changelog.md

```diff
@@ -15,19 +15,44 @@ log][commit-log] for an up-to-date list of raw changes.
 
 - New [resolve topic](https://zulip.com/help/resolve-a-topic) feature
   allows marking topics as completed for lightweight workflow management.
+- New [web-public streams](https://zulip.com/help/web-public-streams)
+  allows Zulip organizations to enable visitors to browse certain
+  streams without creating an account.
```

Comment:
Grammar issue...

Maybe just copy the help center page intro (with a link added)?

> Administrators may enable the option to create web-public streams. Web-public streams can be viewed by anyone on the Internet without creating an account in your organization.